verticaltabs.js

```diff
@@ -807,6 +804,10 @@ VerticalTabs.prototype = {
         this.visibleTabs = Array.filter(this.window.gBrowser.tabs, tab => !tab.hidden && !tab.closing);
         this.filtertabs();
       } else if (purpose === 'tabAction') {
+        if (find_input.value === ''){
```

Comment:
Won't this always be true, because we set `find_input.value = ''` a couple of lines above?